### DISCUSSION TITLE
User counting limitations

### DISCUSSION THREAD

**Discussion by vvcares** [angristan/openvpn-install#931] [Help]
Hi Author,
Im so happy to see this simplified and useful script.
I was used with OVPN Access server free edition.
Is this script have such 2 user count limit as this is not using any Ovpn subscription licenses.

**Comment by toxi22** [comment]
Good afternoon.
No, there is no limit on the number of users.
The number of users depends on the capacity of the virtual server and the bandwidth of the Internet channel.

**Comment by vvcares** [comment]
wow.. Super work then. :-) Best wishes.
